mlflow/protos/webhooks.proto

```diff
@@ -0,0 +1,258 @@
+syntax = "proto2";
+
+package mlflow;
+
+import "scalapb/scalapb.proto";
+import "databricks.proto";
+
+option java_package = "org.mlflow.api.proto";
+option java_generate_equals_and_hash = true;
+option py_generic_services = true;
+option (scalapb.options) = {
+  flat_package: true,
+};
+
+// Webhook service for Model Registry
+service WebhookService {
+  rpc createWebhook (CreateWebhook) returns (CreateWebhook.Response) {
+    option (rpc) = {
+      endpoints: [{
+        method: "POST",
+        path: "/mlflow/webhooks"
```

Comment:
Databricks uses `registry-webhooks` since it's webhooks for model registry.

Reply:
Since we don't need to necessarily follow Databricks, we could add more event types for other resources (experiment, run, ...). [W&B](https://docs.wandb.ai/guides/core/automations/create-automations/webhook/#create-an-automation) supports model registry and webhooks (I'm not saying we should do the same).